src/SWP/Bundle/CoreBundle/Controller/ExternalOauthController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SWP\Bundle\CoreBundle\Controller;
+
+use KnpU\OAuth2ClientBundle\Client\ClientRegistry;
+use Symfony\Component\HttpFoundation\Request;
+use League\OAuth2\Client\Provider\Exception\IdentityProviderException;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ExternalOauthController extends Controller
+{
+    /**
+     * @Route("/connect/oauth", name="connect_oauth_start")
+     */
+    public function connectAction(Request $request)
```

Comment:
please add (where possible) return type hints. 

In this case it will be
```
public function connectAction(Request $request): Response
```

Reply:
Done